src/utils/date.ts

```diff
@@ -0,0 +1,7 @@
+export const getLastYearDate = (date: Date) => {
```

Comment:
p2;

쉽게 유추할 순 있겠지만 return 타입도 명시해주세요~!

```suggestion
export const getLastYearDate = (date: Date): Date => {
```

Reply:
@sohyeonAn 
유틸 함수는 리턴 타입을 명시하면 좋겠네요! 좋은 리뷰 감사합니다아!